### DISCUSSION TITLE
I'm looking for ONNX auto-encoders??

### DISCUSSION THREAD

**Discussion by elephantpanda** [onnx/onnx#4746] [Q&A]
Anyone know where I can get some simple onnx autoencoders?

What I want it for is image interpolation for animations and games.

Image 1----> auto-encode  <----interpolate-----> decode----> image 2

Also, I'm looking for onnx to make "deep dreams". The "onnx zoo" seems a bit limited for generative arts.

**Comment by jcwchen** [comment]
> Also, I'm looking for onnx to make "deep dreams". The "onnx zoo" seems a bit limited for generative arts.

As you mentioned, currently Generative Adversarial Networks (GAN) are not present in ONNX Model Zoo (see the description [here](https://github.com/onnx/models#other-interesting-models-)). Anyone who are interested in contributing GAN models feel free to submit a PR in https://github.com/onnx/models/pulls. Thanks for the contribution.
